Rubberduck.Core/Common/RubberduckHooks.cs

```diff
@@ -18,8 +18,16 @@ public class RubberduckHooks : SubclassingWindow, IRubberduckHooks
         private readonly IList<IAttachable> _hooks = new List<IAttachable>();
         private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
 
+        private static IntPtr GetVbeMainWindowPtr(IVBE vbe)
+        {
+            using (var window = vbe.MainWindow)
+            {
+                return (IntPtr)window.HWnd;
+            }
+        }
+
         public RubberduckHooks(IVBE vbe, IGeneralConfigService config, HotkeyFactory hotkeyFactory, AutoCompleteService autoComplete)
-            : base((IntPtr)vbe.MainWindow.HWnd, (IntPtr)vbe.MainWindow.HWnd)
+            : base(GetVbeMainWindowPtr(vbe), GetVbeMainWindowPtr(vbe))
```

Comment:
I'm not sure I like the double function call here (probably my fault originally) - this would be better with a single argument overload for the ctor that sets both the `hWnd` and the `subclassId` to the passed value.